integration-tests/gatsby-source-wordpress/docker/wordpress/Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM wordpress:5.6
+FROM wordpress:5.9
 
 ARG WP_CONTENT_DIR=/var/www/html/wp-content
 ARG PLUGIN_DIR=${WP_CONTENT_DIR}/plugins
 ARG UPLOADS_DIR=${WP_CONTENT_DIR}/uploads
 
 LABEL maintainer="tyler@gatsbyjs.com"
 
-# Set timezone, install XDebug, PHP Composer, WP-CLI
-RUN echo 'date.timezone = "UTC"' > /usr/local/etc/php/conf.d/timezone.ini \
-  && apt-get update && apt-get install unzip git -y && rm -rf /var/lib/apt/lists/* \
-  && curl -O 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar' \
+# Set timezone
+RUN echo 'date.timezone = "UTC"' > /usr/local/etc/php/conf.d/timezone.ini
+# install system deps
+RUN apt-get update && apt-get install unzip git -y && rm -rf /var/lib/apt/lists/*
```

Comment:
broke these out because one was failing, this way the error message is more helpful